src/POP3.php

```diff
@@ -404,6 +430,26 @@ public function getErrors()
         return $this->errors;
     }
 
+    /**
+     * Set debug output level.
+     *
+     * @param int $level
+     */
+    public function setDebugLevel($level = 0)
+    {
+        $this->do_debug = $level;
```

Comment:
I don't think there is any point on having getters and setters if it's not going to do any validation and the property is public – it's easier to simply use the property directly, as it is in the other classes.

Reply:
Smtp class have same public getter/setter with public do_debug field

Reply:
You'r right! I don't think I've ever seen anyone use it because it's almost always set via the `SMTPDebug` property in the PHPMailer class, and [the one SMTP example](https://github.com/PHPMailer/PHPMailer/blob/master/examples/smtp_check.phps#L22) doesn't use it either. Still, I doubt anyone will be writing new code that uses this. I don't think there's any advantage in changing the API (adding the getter and setter), but changing to using constants for the debug levels is transparent and good practice, so I will accept the PR it if you drop the getter and setter and leave the constants in.

Reply:
Fixed